cli/bblfshctl/cmd/driver_install.go

```diff
@@ -21,17 +21,17 @@ var (
 		"python":     "docker://bblfsh/python-driver:latest",
 		"java":       "docker://bblfsh/java-driver:latest",
 		"bash":       "docker://bblfsh/bash-driver:latest",
-		"php":        "docker://bblfsh/php-driver:latest",
```

Comment:
It think should remain in this list (see JS/Ruby/Python)

Reply:
True!